### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by zann1x**
In VulkanImGui.cpp there was a misspelled local variable

### CODE REVIEW

**Comment by ScrappyCocco** [comment]
Uh thank you so much! 
Since you edited that file and i just noticed that the staging buffer is a standard pointer,
could you edit it to `unique_ptr` ? Similar as it's done in [here in the VertexBuffer](https://github.com/ScrappyCocco/ScrapEngine/blob/master/ScrapEngine/ScrapEngine/Engine/Rendering/Buffer/VertexBuffer/VertexBuffer.cpp#L11)
It's used only in that function so it's kinda bad to see to manually delete the pointer
(I should use smart pointers in more places but this is a beginning)

**Comment by ScrappyCocco** [comment]
(Oh also feel free to just open an issue linking the line if it's just a typo, so you don't need to clone the whole thing unless you want to)

**Comment by zann1x** [comment]
There you go :)
Looking through the code I also noticed that you imported <memory> in VulkanSimpleMaterialPool.cpp while it was already being imported in the header file, so I removed that as well.

File: ScrapEngine/ScrapEngine/Engine/Rendering/Model/ObjectPool/VulkanSimpleMaterialPool/VulkanSimpleMaterialPool.cpp [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #include <Engine/Rendering/Model/ObjectPool/VulkanSimpleMaterialPool/VulkanSimpleMaterialPool.h>
-#include <memory>
```
Comment (ScrappyCocco) [inline ScrapEngine/ScrapEngine/Engine/Rendering/Model/ObjectPool/VulkanSimpleMaterialPool/VulkanSimpleMaterialPool.cpp]: Oh thank you so much!

**Pull request merged**